### DISCUSSION TITLE
[Feature] Open Highlight on Hover over of unrecognized face

### DISCUSSION THREAD

**Discussion by diablo581** [immich-app/immich#11173] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Often it is hard to tell who a face is from the little square in the person highlight.  If it was possible to pop up the image the highlight was taken from it might make it easier to quickly address some of the unknown people. Currently you have to click into the image, realize it isn't a person you know, then back out and hide them. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by daFreeMan** [comment]
Came here to request this. Leaving disappointed that it's been more than a year and it hasn't even been commented on, to say anything about being implemented.

**Reply by bo0tzz** [reply]
@daFreeMan there's thousands of feature requests, if we went around leaving comments on each of them that'd take away a meaningful amount of time that could be spent implementing them instead.
